Ginger/GingerCoreNET/ActionsLib/UI/Web/ActAccessibilityTesting.cs

```diff
@@ -824,15 +824,12 @@ public void SetAxeResultToAction(AxeResult axeResult)
         {
             bool hasAnyViolations = axeResult.Violations.Any();
             bool ActionResult = true;
-
+            List<string> AcceptableSeverity = new List<string>();
```

Comment:
Please use Enumerable.Empty<string>().ToList(); to save memory instead of new List<string>();

Reply:
done